pkg/controller/operators/catalog/operator_test.go

```diff
@@ -704,17 +705,18 @@ func NewFakeOperator(ctx context.Context, namespace string, watchedNamespaces []
 				// 1 qps, 100 bucket size.  This is only for retry speed and its only the overall factor (not per item)
 				&workqueue.BucketRateLimiter{Limiter: rate.NewLimiter(rate.Limit(1), 100)},
 			), "resolver"),
-		sources:               make(map[resolver.CatalogKey]resolver.SourceRef),
-		resolver:              config.resolver,
+		resolver:              &fakes.FakeResolver{},
```

Comment:
I think `config.resolver` is the value we want here. It's defaulted to `&fakes.FakeResolver{}` above and allows test authors to specify a specific fake using the `withResolver(...)` option.

Reply:
Ah, yep - this flipped when I rebased